### PR TITLE
Make intake run during button press

### PR DESCRIPTION
**Pull request by ChainedGCH**
Makes the intake run during a button press

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -51,6 +62,17 @@ public RobotContainer() {
   private void configureBindings() {
     new Trigger(() -> triggerPressed())
         .whileTrue(new BasicDriveCommand(m_robotDrive, m_driverController));
+
+    // TODO: make actually intake under button press
+    new Trigger(this::buttonPressed)
+        .whileTrue(new RunIntake(m_intake));
+  }
+
+  public boolean buttonPressed() {
```
Comment (EmDash00) [inline src/main/java/frc/robot/RobotContainer.java]: Which button? The controller has many.

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -51,6 +62,17 @@ public RobotContainer() {
...
+    if (m_operatorController.getRawButton(Bindings.intakeNote)) {
```
Comment (EmDash00) [inline src/main/java/frc/robot/RobotContainer.java]: STUDENTS! Stop using if statements if you're returning a boolean...

Cease this:
```java
if (condition) {
	return true;
}
else {
	return false;
}
```

Be a good programmer. Be responsible. Do this instead:
```java
return condition;
```

File: src/main/java/frc/robot/constants/RobotConstants.java [hunk 3]
```diff
@@ -123,4 +123,8 @@ public static final class IntakeConstants {
     // TODO: update placeholder roller Motor ID
     public static final int kMotorID = -1;
   }
+
+  public class Bindings {
```
Comment (EmDash00) [inline src/main/java/frc/robot/constants/RobotConstants.java]: What kind of bindings?

**Review by EmDash00 — DISMISSED** [review]

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (ChainedGCH) [reply src/main/java/frc/robot/RobotContainer.java]: Responded to

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (ChainedGCH) [reply src/main/java/frc/robot/RobotContainer.java]: Responded to

File: src/main/java/frc/robot/constants/RobotConstants.java (hunk 3 above)
Comment (ChainedGCH) [reply src/main/java/frc/robot/constants/RobotConstants.java]: Responded to

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (EmDash00) [reply src/main/java/frc/robot/RobotContainer.java]: I mean that the function should be self-descriptive. Javadoc your functions as well.

File: src/main/java/frc/robot/constants/RobotConstants.java (hunk 3 above)
Comment (EmDash00) [reply src/main/java/frc/robot/constants/RobotConstants.java]: The class should be self describing. Call it `ButtonBindings` and make a javadoc for it.

**Review by EmDash00 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/constants/RobotConstants.java (hunk 3 above)
Comment (EmDash00) [reply src/main/java/frc/robot/constants/RobotConstants.java]: Also move to `RobotConfig` and call it `IntakeBindings`. Also constants should be prefixed with `k`

File: src/main/java/frc/robot/constants/RobotConfig.java [hunk 4]
```diff
@@ -34,4 +34,9 @@ public static final class IntakeConfig {
     // TODO: update timeout time (in seconds)
     public static final double kRunIntakeTimeoutSecs = 0;
   }
+
+  // Button bindings (There is other code on the button bindings branch)
```
Comment (EmDash00) [inline src/main/java/frc/robot/constants/RobotConfig.java]: Delete this comment.

**Review by EmDash00 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/constants/RobotConfig.java (hunk 4 above)
Comment (ChainedGCH) [reply src/main/java/frc/robot/constants/RobotConfig.java]: Done

File: src/main/java/frc/robot/constants/RobotConfig.java [hunk 5]
```diff
@@ -34,4 +34,8 @@ public static final class IntakeConfig {
     // TODO: update timeout time (in seconds)
     public static final double kRunIntakeTimeoutSecs = 0;
   }
+
+  public class IntakeBindings {
+    public static final int kIntakeNote = 2;
```
Comment (EmDash00) [inline src/main/java/frc/robot/constants/RobotConfig.java:39]: Call it `kIntakeNoteButton`

**Review by EmDash00 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**